.eslintrc.json

```diff
@@ -15,8 +15,9 @@
     "no-param-reassign": "off",
     "eol-last": "off",
     "import/extensions": [ 1, {
-      "js": "always", "json": "always"
-    }]
+      "js": "off", "json": "always"
+    }],
+    "array-callback-return": "off"
```

Comment:
- [ ] Please make sure you dont alter, modify or disable any of the linter configurations because it is setup to ensure you follow the best coding practices 👍🏽